kmd.tree

```diff
@@ -22,9 +22,12 @@
         <toc-element id="organize-process-around-kmm.md"/>
         <toc-element id="architect-kmm-app.md"/>
         <toc-element id="connect-to-platform-specific-apis.md"/>
-        <toc-element id="configure-sqldelight-for-data-storage.md"/>
-        <toc-element id="use-ktor-for-networking.md"/>
         <toc-element id="add-dependencies.md"/>
+        <toc-element toc-title="Networking and data storage">
+            <toc-element id="configure-sqldelight-for-data-storage.md"/>
+            <toc-element id="use-ktor-for-networking.md"/>
+            <toc-element id="hands-on-networking-data-storage.md"/>
```

Comment:
Its title is also not consistent with others. Please update by adding a verb or rephrasing.